src/ReactiveUI.Winforms/ReactiveUI.Winforms.csproj

```diff
@@ -1,12 +1,12 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras">
 
   <PropertyGroup>
-    <TargetFrameworks>net461</TargetFrameworks>
+    <TargetFramework>net461</TargetFramework>   
```

Comment:
Use TargetFrameworks. MsBuild.Sdk.Extras require it. Also there is fundamentally no difference apart from the ability to be recursive with checks.